### PR TITLE
update ocs4.adoc and add new images

### PR DESCRIPTION
**Pull request by netzzer**
Content modified to this section:
https://github.com/red-hat-storage/ocs-training/tree/ocs42-fix4/ocp4ocs4#creating-an-object-bucket-claim

### CODE REVIEW

**Comment by mulbc** [comment]
Resolves #61

**Comment by mulbc** [comment]
Resolves #66

File: ocp4ocs4/ocs4.adoc [hunk 1]
```diff
@@ -133,11 +142,13 @@ cluster-ocs-79cf-lj8wq-workerocs-us-east-1b-jh6k4   running   m4.4xlarge   us-ea
...
 
-You can see that the workerocs *machines* are using a different AWS EC2 instance type `m4.4xlarge`. The `m4.4xlarge` instance type follows our recommended instance sizing for OCS and provides the best performance/price ratio. Now you want to see if our new *machines* are added to the OCP cluster.
+You can see that the workerocs *machines* are using are also using the AWS EC2 instance type `m4.4xlarge`. The `m4.4xlarge` instance type follows our recommended instance sizing for OCS, 16 cpu and 64 GB mem. 
+
+Now you want to see if our new *machines* are added to the OCP cluster.
 
 [source,role="execute"]
 ----
-watch oc get machinesets -n openshift-machine-api
+oc get machinesets -n openshift-machine-api -w
```
[the remark this answers is not in the document]
Comment (netzzer) [reply ocp4ocs4/ocs4.adoc]: I found 'watch' to not be available on my macbook.

**Comment by mulbc** [comment]
Discussed with Annette and `watch` is not preinstalled on Macs so we change to `-w`

**Pull request merged**